Changed 'people' to 'accounts' (#765)
github/docs: content/github/finding-security-vulnerabilities-and-errors-in-your-code/managing-code-scanning-alerts-for-your-repository.md

shortTitle: Managing alerts
intro: 'You can view, fix, dismiss, or delete alerts for potential vulnerabilities or errors in your project''s code.'
product: '{% data reusables.gated-features.code-scanning %}'

> permissions: 'Accounts with write permission to a repository can manage {% data variables.product.prodname_code_scanning %} alerts for the repository.'

Reviewer: You're right of course that it's accounts that are assigned permissions. However, I think saying that "accounts ... can manage ..." isn't quite right either. We could say something like "If you have an account that has write permission ... you can manage ..." but it's a little long-winded and I think it would be more in keeping with our conversational style (while not being easily misunderstood) just to say:
```suggestion
permissions: 'If you have write permission to a repository you can manage {% data variables.product.prodname_code_scanning %} alerts for that repository.'
```

Reviewer 2: 3fdb22b84fb3eb5dd870e09b0997dc95d3e2b51b
